endpoints/setuid.go

```diff
@@ -73,6 +75,19 @@ func NewSetUIDEndpoint(cfg config.HostCookie, syncersByBidder map[string]usersyn
 			return
 		}
 
+		accountID := query.Get("account")
+		if accountID == "" {
+			accountID = metrics.PublisherUnknown
+		}
+		_, fetchErrs := accountService.GetAccount(context.Background(), cfg, accountsFetcher, accountID)
```

Comment:
For my knowledge, is this variable with the name `_` the spot where the fetched account will be utilized in the future GDPR PR that you mentioned in the description?

Reply:
Yes, that's correct; that underscore will become `account` in a future PR. I'll then pass `account` into `gdpr.NewTCF2Config` which will create an aggregated GDPR config object consisting of the host and account GDPR settings that can then be used to build a GDPR permissions object.

Reply:
I added a comment so this is evident.